internal/protocoltest/jsonrpc/types/enums.go

```diff
@@ -0,0 +1,13 @@
+// Code generated by smithy-go-codegen DO NOT EDIT.
+package types
```

Comment:
Tried rebuilding locally, and this file seems to be deleted. Does it have to do with updating smithy version?

Reply:
Yeah you would need to build smithy from master. That and all the error tests are still in pr: https://github.com/awslabs/smithy/pull/484

Reply:
Update: that pr is merged, so just going by master should be good